freeipa/src/main/resources/freeipa-salt/pillar/upgradeccm/init.sls

```diff
@@ -8,3 +8,4 @@ ccm_jumpgate:
   environment_crn:
   agent_access_key_id: #only for one-way TLS
   agent_enciphered_access_key: #only for one-way TLS
+  activation_in_minutes:
```

Comment:
from CDP clusters we already know that empty pillars are not necessary usually, as they cause more issue if the bootstrap part happens, but the pillars can't be pushed down. could you revisit if this empty pillar necessary at all and remove if not

Reply:
removed it